common-rest/src/main/java/org/uniprot/api/common/repository/search/facet/FacetConverter.java

```diff
@@ -1,14 +1,14 @@
 package org.uniprot.api.common.repository.search.facet;
 
-import java.util.List;
+import java.util.Collection;
 import java.util.Map;
 import java.util.Optional;
 
 import org.uniprot.core.util.Utils;
 
 public abstract class FacetConverter<F, T> {
 
-    public abstract T convert(F queryResponse, List<String> facetList);
+    public abstract T convert(F queryResponse, Collection<String> facetList);
 
```

Comment:
I couldn't understand the need to change the 2nd parameter from List to Collection.